include/config/SkUserConfig.h

```diff
@@ -100,11 +100,10 @@
  */
 //#define SK_MAX_SIZE_FOR_LCDTEXT     48
 
-// [ivan4258] For RTC, we always use BGRA.
-#define SK_B32_SHIFT 0
-#define SK_G32_SHIFT 8
-#define SK_R32_SHIFT 16
-#define SK_A32_SHIFT 24
+/*  Change the kN32_SkColorType ordering to BGRA to work in X windows.
+ */
+//#define SK_R32_SHIFT    16
```

Comment:
Can we just remove this commented-out `#define`?

Reply:
This is actually restoring the upstream text on this file. I migrated all RTC defines to the end of the file to reduce the possibility of upstream merge conflicts in the future and to centralize all of our edits.